src/marketplace/MarketplaceCard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Card, CardTitle, CardBody, CardFooter, Label, Button } from '@patternfly/react-core';
+import { DatabaseIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
+import ImageWithPlaceholder from '../components/TilesShared/ImageWithPlaceholder';
+
+const MarketplaceCard = (product) => (
+  <Card isFlat className="marketplace-card">
+    <CardTitle className="pf-u-pl-md pf-u-pr-md">
+      <ImageWithPlaceholder src={product.publishedContent.logo.url} style={{ maxHeight: 24 }} />
```

Comment:
Not sure how the styling is handled, but don't we want to have styles in scss files instead?